pybaseball/datahelpers/transform.py

```diff
@@ -1,21 +1,21 @@
 
 import pandas as pd
 
-from pybaseball.utils import NULLABLE_INT
 from pybaseball.lahman import people
 
 
-def get_age(stats_df, people_df=None):
+def get_age(stats_df: pd.DataFrame, people_df: pd.DataFrame = None) -> pd.DataFrame:
```

Comment:
MyPy doesn't complain on this. I believe this is because I believe classes (as opposed to internal types) are Optional by default. I can add it for clarity though if we'd prefer.